meshkernel/meshkernel.py

```diff
@@ -88,16 +81,69 @@ def __init__(self, is_geographic: bool = False):
         elif system == "Darwin":
             lib_path = os.path.join(file_path, "libMeshKernelApi.dylib")
         else:
-            if not str:
+            if not system:
                 system = "Unknown OS"
             raise OSError("Unsupported operating system: {}".format(system))
 
         self.lib = CDLL(str(lib_path))
+        self.ExitCode = self.__get_exit_codes()
```

Comment:
Sonar Cloud complains about ExitCode name. Should it be   self.exit_code ?

Reply:
done